packages/belos/tpetra/test/TFQMR/test_tfqmr_hb.cpp

```diff
@@ -40,52 +40,68 @@
 //@HEADER
 //
 // This driver reads a problem from a Harwell-Boeing (HB) file.
+// The right-hand-side from the HB file is used instead of random vectors.
 // The initial guesses are all set to zero.
 //
 // NOTE: No preconditioner is used in this case.
 //
-#include "BelosConfigDefs.hpp"
-#include "BelosLinearProblem.hpp"
-#include "BelosTpetraAdapter.hpp"
-#include "BelosTFQMRSolMgr.hpp"
-#include "BelosPseudoBlockTFQMRSolMgr.hpp"
-#include "BelosTpetraTestFramework.hpp"
 
+// Tpetra
+#include <Tpetra_Core.hpp>
+#include <Tpetra_CrsMatrix.hpp>
+#include <Tpetra_Map.hpp>
+
+// Teuchos
 #include <Teuchos_CommandLineProcessor.hpp>
 #include <Teuchos_ParameterList.hpp>
 #include <Teuchos_StandardCatchMacros.hpp>
-#include <Tpetra_Core.hpp>
-#include <Tpetra_CrsMatrix.hpp>
 
+// Trilinos
+#include <Trilinos_Util.h>
 
-int main(int argc, char *argv[]) {
-  //
-  typedef Tpetra::MultiVector<>::scalar_type ST;
-  typedef Teuchos::ScalarTraits<ST>       SCT;
-  typedef SCT::magnitudeType               MT;
-  typedef Tpetra::Operator<ST>             OP;
-  typedef Tpetra::MultiVector<ST>          MV;
-  typedef Belos::OperatorTraits<ST,MV,OP> OPT;
-  typedef Belos::MultiVecTraits<ST,MV>    MVT;
+// Belos
+#include "BelosConfigDefs.hpp"
+#include "BelosLinearProblem.hpp"
+#include "BelosTpetraAdapter.hpp"
+#include "BelosTFQMRSolMgr.hpp"
+#include "BelosPseudoBlockTFQMRSolMgr.hpp"
+#include "BelosTpetraTestFramework.hpp"
```

Comment:
`BelosTpetraTestFramework.hpp` also includes `BelosTpetraAdapter.hpp` and `Trilinos_Util_iohb.h` (you don't have to explicitly include them)

Reply:
Fix, thanks!